.github/workflows/cd.yml

```
@@ -0,0 +1,36 @@
+    workflows:
+      - 'Go CI/CD Pipeline'
+    branches:
+      - 'master'
+    types:
+      - 'completed'
+
+jobs:
+  deploy:
+    name: Push new image into registry
+    runs-on: ubuntu-latest
+    if: ${{github.event.workflow_run.conclusion == 'success'}} &&
```

actionlint (1.7.4)

if: condition "${{github.event.workflow_run.conclusion == 'success'}} && ${{contains(toJson(github.event.workflow_run.head_commit.modified), 'version')}}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)